### DISCUSSION TITLE
Personalized search results

### DISCUSSION THREAD

**Discussion by vinc3nz** [spotify-web-api-java/spotify-web-api-java#404] [Q&A] [answered]
I have an application that allows searching for songs. Let's say the users searches for "test" on his PC, other results show up that if he does in my app. Is there any way to get the same/similar results as if he would search on his account?

**Comment by Maskowh** [comment, marked as answer]
Hey, here is the request sent to Spotify when I search "test" on the desktop client.
As you can see, this is not the public endpoint that we have access to through the Spotify public API. Some params seems to be different or not available on the public API, and I'm pretty sure the two endpoints don't work the same way, so I'm afraid you won't be able to have the exact same results, but maybe you can have closer ones.

![image](https://github.com/user-attachments/assets/51c0e881-446c-470e-a120-34e11d8703fb)

![image](https://github.com/user-attachments/assets/3c3c491f-2028-49b1-8169-9e38e6340152)

**Reply by vinc3nz** [reply]
Thank you very much for your reply, I will have a closer look into this.
